charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -20,6 +20,7 @@ spec:
         release: "{{ .Release.Name }}"
         heritage: "{{ .Release.Service }}"
     spec:
+      serviceAccountName: controller
```

Comment:
I see you added defaults to `charts.yaml`, but you still need to templatize the definition here and in `apiserver-deployment.yaml`.